src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -989,5 +991,45 @@ private void VerifyActiveProxy()
         {
             ErrorUtilities.VerifyThrow(_activeProxy, "Attempted to use an inactive task host.");
         }
+
+        /// <summary>
+        /// Inialize new RAR node
+        /// </summary>
+        bool IRarBuildEngine.CreateRarNode()
```

Comment:
This is a task creating a RarNode? Does that make sense?

Reply:
I need to start local node from RAR task. I haven't discovered any mechanism which would allow me that, if there is any, I will change it.

Reply:
Related to this: https://github.com/dotnet/msbuild/pull/5613/files#r477380003. Depending on result of that, we could remove this code.